modules/observability/grafana/.bicep/nested_privateEndpoint.bicep

```diff
@@ -0,0 +1,58 @@
+param location string
```

Comment:
Can you update the path of the child module file? It would be better if you can put it under `/grafana/modules/`. We just updated the contribution doc to mention that: https://github.com/Azure/bicep-registry-modules/blob/main/CONTRIBUTING.md#nested-bicep-files.

Reply:
done!